web/sites/default/config/webform.webform.residential_discount_calculator.yml

```diff
@@ -0,0 +1,295 @@
+    cisterns:
+      '#type': checkbox
+      '#title': 'From the roof direct to cisterns or rain barrels that allow for overflow onto my property'
+    pond:
+      '#type': checkbox
+      '#title': 'From the roof to a holding facility, stormwater planter or pond that slowly releases stormwater off my property'
+  other_on_site_stormwater_management:
+    '#type': fieldset
+    '#title': 'Other kinds of on-site stormwater management on your property'
+    '#description': '<p>Check all of the following conditions that apply to your property.</p>'
+    '#description_display': before
+    ecoroof:
+      '#type': checkbox
+      '#title': 'I have installed an ecoroof on a portion or all of my buildings'
+    impervious_area:
+      '#type': checkbox
+      '#title': 'The footprint (or total area) of buildings, driveways, patios, walkways, or other hard surfaces on my property cover less than 1,000 square feet'
+    trees:
+      '#type': checkbox
+      '#title': 'There are 4 or more trees on my property that are taller than 15 feet'
+      '#description': '<p>This does not include trees in the landscaped parking strip or public-right-of-way.</p>'
+  calculate:
+    '#type': webform_actions
+    '#title': 'Submit button(s)'
+    '#submit__label': 'Calculate discount'
+    '#submit__attributes':
+      class:
+        - mb-8
+  monthly_stormwater_management_charges:
+    '#type': fieldset
+    '#title': 'Monthly Stormwater Management Charges'
+    streets:
+      '#type': textfield
+      '#title': 'Off-site:'
+      '#title_display': inline
+      '#size': 8
+      '#readonly': true
+    onsite:
+      '#type': textfield
+      '#title': 'On-site:'
+      '#title_display': inline
+      '#size': 8
+      '#readonly': true
+    gross_charge:
+      '#type': textfield
+      '#title': 'Total:'
+      '#title_display': inline
+      '#size': 8
+      '#readonly': true
+  stormwater_discount:
+    '#type': fieldset
+    '#title': 'Stormwater Discount'
+    '#description': '<p>This section shows how your discount will be applied to your utility bill. The discount is pro-rated on a quarterly basis depending on the length of your utility billing cycle.</p>'
+    '#description_display': before
+    discount:
+      '#type': textfield
+      '#title': 'Clean River Reward Discount:'
+      '#help': "<p>[This text is overwritten by the webform's calculator JavaScript code.]</p>"
+      '#title_display': inline
+      '#help_display': element_after
+      '#size': 8
+      '#readonly': true
+    net_charge:
+      '#type': textfield
+      '#title': 'Adjusted Stormwater Management Charge:'
+      '#title_display': inline
+      '#size': 8
+      '#readonly': true
+      '#wrapper_attributes':
+        class:
+          - bg-light
+  footer_text:
+    '#type': webform_markup
+    '#markup': '<p>If you have questions about how to register or how the discount is calculated, please call (503) 823-1371.</p>'
```

Comment:
Can we make this 503-823-1371 with a tel link, to match our standard format?

Otherwise, looks good! Nice work.